src/azure-cli/azure/cli/command_modules/security/_help.py

```diff
@@ -90,6 +90,11 @@
         az security atp storage update --resource-group MyResourceGroup --storage-account MyStorageAccount --is-enabled false
 """
 
+helps['security va'] = """
+type: group
+short-summary: View Vulnerability Assessment.
+"""
+
```

Comment:
Please split it to a separate PR or merge from `dev` to incorporate the change.

Reply:
If I split to other PR, it will fail in keyvault test package.
For now, `dev` doesn't cover this help fix.

This is already a split PR fixed for #18868. Just want to make things simple.

Reply:
merged with `dev`